go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -688,8 +689,9 @@ func (nc *DefaultNodeNetworkController) Start(ctx context.Context) error {
 	nc.wg.Add(1)
 	go func() {
 		defer nc.wg.Done()
-		nc.routeManager.Run(nc.stopChan)
```

Comment:
no reason besides thinking 2 mins was overly aggressive re-sync period especially considering routes can be restored instantly anyway because we listen to events.